lib/utils.js

```diff
@@ -677,7 +675,7 @@ exports.stackTraceFilter = function() {
       }
 
       // Clean up cwd(absolute)
-      if (/\(?.+:\d+:\d+\)?$/.test(line)) {
+      if (/:\d+:\d+\)?$/.test(line)) {
         line = line.replace('(' + cwd, '(');
```

Comment:
Or tweak the original regex so first line won't match immediately...
```js
      if (/^ \(?.+:\d+:\d+\)?$/.test(line)) {
```

Reply:
afaik lines displayed after the message don't necessary start with a space, or whitespace at all.  even though Node.js may display stacktraces this way (is it four spaces or a tab??), `stack` is a non-standard property, and implementations differ.

Reply:
While unspecified, the stacktrace itself _is_ indented in all viable implementations.
 ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) say MS doesn't have `Error.stack`).
```js
      if (/^\s\(?.+:\d+:\d+\)?$/.test(line)) {
```

Just spitballing here...